pinot-common/src/main/java/org/apache/pinot/common/metadata/segment/SegmentZKMetadata.java

```diff
@@ -410,21 +408,4 @@ public TimeUnit getTimeUnit() {
       return null;
     }
   }
-
```

Comment:
This code is deprecated and it doesn't look like is used anywhere in Pinot - I've checked for the string version of the method names in case reflection was used. I don't think projects other than Pinot should be using this class, I've searched in both Trino and Presto and the class doesn't seem to be in use